pallets/communities/src/impls.rs

```diff
@@ -0,0 +1,26 @@
+use crate::{Communities, CommunityId, Config};
+use orml_payments::{FeeHandler, FeeRecipientShareList, PaymentDetail};
+use sp_std::str::FromStr;
+
+/// Struct to implement community fee handler
+pub struct CommunityFeeHandler;
+
+/// FeeHandler trait implementation for Communities
+impl<T: Config> FeeHandler<T> for CommunityFeeHandler {
+	fn apply_fees(
+		_from: &T::AccountId,
+		_to: &T::AccountId,
+		_detail: &PaymentDetail<T>,
+		remark: Option<&[u8]>,
+	) -> FeeRecipientShareList<T> {
+		if let Some(remark) = remark {
```

Comment:
A more useful initial implementation could be to check the validity of the community_id after parsing, if invalid let's charge an arbitrary amount(e.g. 1%) that goes to the pallet id turned into an account(just some random account that is not accessible to users) and if valid and the community is registered then we turn the community_id [into an account](https://paritytech.github.io/substrate/master/sp_runtime/traits/trait.AccountIdConversion.html) that would receive a fee lower than that of an invalid remark(e..g 0.5%)